setup.py

```diff
@@ -108,6 +108,12 @@
 
     cmdclass=versioneer.get_cmdclass(),
 
+    dependency_links = [
+        # Use our fork of testtools until #165, #171, and #172 are merged and
+        # released. See FLOC-3498.
+        "git+git://github.com/ClusterHQ/testtools@clusterhq-fork#egg=testtools-1.8.2chq1",  # noqa
```

Comment:
is git+git a thing? It's a hard thing to google search for so I couldn't verify.